integration-tests/exporter/exporter_test.go

```diff
@@ -0,0 +1,242 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openebs/maya/integration-tests/artifacts"
+	framework "github.com/openebs/maya/integration-tests/framework/v1alpha1"
+	k8s "github.com/openebs/maya/pkg/client/k8s/v1alpha1"
+	cv "github.com/openebs/maya/pkg/cstorvolume/v1alpha1"
+	pvc "github.com/openebs/maya/pkg/kubernetes/persistentvolumeclaim/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	// namespaceYaml holds the yaml spec
+	// to create admission namespace
+	namespaceYaml artifacts.Artifact = `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: pool-exporter
+`
+	// cStorPVC holds the yaml spec
+	// for source persistentvolumeclaim
+	cStorPVC artifacts.Artifact = `
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: cstor-source-volume
+  namespace: pool-exporter
+  labels:
+    name: cstor-source-volume
+spec:
+  storageClassName: openebs-cstor-class
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: "2G"
+`
+	// singleReplicaSC holds the yaml spec
+	// for pool with single replica
+	singleReplicaSC artifacts.Artifact = `
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-cstor-class
+  annotations:
+    cas.openebs.io/config: |
+      - name: StoragePoolClaim
+        value: "cstor-sparse-pool"
+      - name: ReplicaCount
+        value: "1"
+    openebs.io/cas-type: cstor
+provisioner: openebs.io/provisioner-iscsi
+reclaimPolicy: Delete
+`
+	// poolService holds the yaml spec
+	// for creating service for cstor-pool deployment
+	poolService artifacts.Artifact = `
+apiVersion: v1
+kind: Service
+metadata:
+  name: cstor-pool-svc
+  namespace: pool-exporter
+  labels:
+    app: cstor-pool
+    openebs.io/storage-pool-claim: cstor-sparse-pool
+spec:
+  ports:
+  - port: 9500
+    targetPort: 9500
+  selector:
+    app: cstor-pool
+`
+)
+
+var _ = Describe("[single-node] Pool exporter", func() {
+	options := framework.FrameworkOptions{
+		MinNodeCount: 1,
+		Artifacts:    "../artifacts/openebs-ci.yaml",
+	}
+	_ = framework.NewFrameworkDefault("Pool exporter test", options)
+
+	var (
+		NSUnst, SCUnst, PVCUnst, PoolSVCUnst *unstructured.Unstructured
+		pvclaim                              *corev1.PersistentVolumeClaim
+		poolSvc                              *corev1.Service
+	)
+	BeforeEach(func() {
+		// Extracting storageclass artifacts unstructured
+		var err error
+		SCUnst, err = artifacts.GetArtifactUnstructured(singleReplicaSC)
+		Expect(err).ShouldNot(HaveOccurred(), "while parsing storage class yaml: ", singleReplicaSC)
+
+		// Apply  single replica storageclass
+		cu := k8s.CreateOrUpdate(
```

Comment:
These packages are going to be deprecated. 
Lets make use of individual `pkg/kubernetes/<nativeresource>` or `pkg/<customresource>` builders & kubeclients.